Maple2.Server.Game/Manager/Items/EquipManager.cs

```diff
@@ -193,6 +203,26 @@ public class EquipManager {
         }
     }
 
+    /// <summary>
+    /// Unequips an cosmetic item (Hair, Eyes, Face Decor) and discards the previous cosmetic item.
+    /// </summary>
+    public bool EquipCosmetic(Item cosmetic, EquipSlot equipSlot) {
```

Comment:
missing validation:
```cs
if (slot is not (EquipSlot.HR or EquipSlot.FA or EquipSlot.FD)) {
    return false;
}

if (!ValidEquipSlotForItem(slot, item)) {
    return false;
}
````

maybe this as well, but it might be unnecessary:
```cs
StringCode result = ValidateEquipItem(session, item);
if (result != StringCode.s_empty_string) {
    session.Send(NoticePacket.MessageBox(result));
    return false;
}
```